VimeoNetworking/Sources/Models/Album.swift

```diff
@@ -0,0 +1,168 @@
+        }
+        
+        struct Value {
+            static let Name = "albumName"
+            static let Description = "albumDescription"
+            static let Logo = "albumLogo"
+            static let CreatedTime = "createdTimeString"
+            static let ModifiedTime = "modifiedTimeString"
+            static let VideoThumbnails = "videoThumbnails"
+            static let Password = "password"
+        }
+        
+        struct Class {
+            static let PictureCollection = VIMPictureCollection.self
+            static let Privacy = VIMPrivacy.self
+            static let User = VIMUser.self
+            static let Embed = AlbumEmbed.self
+            static let MutableDictionary = NSMutableDictionary.self
+        }
+        
+        struct Collections {
+            static let Pictures = "pictures"
+            static let Array = NSArray.self
+        }
+    }
+    
+    @objc public var albumName: String?
+    @objc public var albumDescription: String?
+    @objc public var albumLogo: VIMPictureCollection?
+    @objc public var createdTimeString: String?
+    @objc public var createdTime: Date?
+    @objc public var modifiedTimeString: String?
+    @objc public var modifiedTime: Date?
+    @objc public var privacy: VIMPrivacy?
+    @objc public var duration: NSNumber?
+    @objc public var uri: String?
+    @objc public var link: String?
+    @objc public var embed: AlbumEmbed?
+    @objc public var videoThumbnails: NSArray?
+    @objc public var user: VIMUser?
+    @objc public var theme: String?
+    @objc dynamic private var metadata: [String: Any]?
+    @objc dynamic private var connections: [String: Any]?
+    
+    public override func getObjectMapping() -> Any! {
+        return [
+            Constant.Key.Name : Constant.Value.Name,
+            Constant.Key.Description : Constant.Value.Description,
+            Constant.Key.Logo : Constant.Value.Logo,
+            Constant.Key.CreatedTime : Constant.Value.CreatedTime,
+            Constant.Key.ModifiedTime : Constant.Value.ModifiedTime,
+            Constant.Key.Pictures : Constant.Value.VideoThumbnails
+        ]
+    }
+    
+    public override func getClassForObjectKey(_ key: String!) -> AnyClass? {
+        switch key {
+        case Constant.Key.User:
+            return Constant.Class.User
+        case Constant.Key.Privacy:
+            return Constant.Class.Privacy
+        case Constant.Key.Logo:
+            return Constant.Class.PictureCollection
+        case Constant.Key.Embed:
+            return Constant.Class.Embed
+        case Constant.Key.Metadata:
+            return Constant.Class.MutableDictionary
+        default:
+            return nil
+        }
+    }
+    
+    public override func getClassForCollectionKey(_ key: String!) -> AnyClass! {
+        switch key {
+        case Constant.Collections.Pictures:
+            return Constant.Class.PictureCollection
+        default:
+            return nil
```

Comment:
Looks like if we can return nil should this be an optional return value?
Force unwrapping needed on input param?